cpp/arcticdb/entity/data_error.hpp

```diff
@@ -9,8 +9,6 @@
 
 #include <string>
 
-#include <fmt/format.h>
-
 #include <arcticdb/pipeline/query.hpp>
```

Comment:
Looks like `<arcticdb/pipeline/query.hpp>` is not needed anymore.

Reply:
It is, for `VersionQueryType`